CAIPs/caip-draft_cacao.md

```diff
@@ -70,20 +71,26 @@ It is important to note, that issuer here is [did:pkh](https://github.com/spruce
 Also, as per [EIP-4361 "Sign-in with Ethereum"](https://github.com/ethereum/EIPs/blob/5e9b0fe0728e160f56dd1e4cbf7dc0a0b1772f82/EIPS/eip-4361.md) specificaction,
 `iat`, `nbf`, and `exp` are encoded as [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) `date-time`, which could include milliseconds precision.
 
-The difference is how we do signature verification.
+The signature in essence is just bytes, but we have to give a hint on how the signature verification should work.
+At the moment, we limit the signature verification by two types:
+- `eip191` indicates that that signature is made by an Ethereum [externally owned account](https://www.ethdocs.org/en/latest/contracts-and-transactions/account-types-gas-and-transactions.html#externally-owned-accounts-eoas) (EOA),
+- `eip1271` indicates that the signature is made by an Etereum [contract account](https://www.ethdocs.org/en/latest/contracts-and-transactions/account-types-gas-and-transactions.html#contract-accounts) (like Gnosis Safe or Argent); the verification should be done according to [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271).
 
-The signature in essence is just bytes, but we also would like to provide additional meta-information at a later time.
+In the future, we anticipate creating a specialized registry for signature types.
 
 ```
 type Signature struct {
-  m optional SignatureMeta // For future extension
+  m optional SignatureMeta
   s Bytes
 }
 
 type SignatureMeta struct {
+    t String //= "eip191" or "eip1271"
```

Comment:
Can we make `t` a  required property of the `Signature` type? It seems like it will always be needed?

Reply:
Seems like this approach was preferred by @pedrouid also (see above)

Reply:
See my new comment with suggested change.